src/python_testing/TC_TIMESYNC_2_2.py

```diff
@@ -29,9 +29,6 @@ async def read_ts_attribute_expect_success(self, endpoint, attribute):
         cluster = Clusters.Objects.TimeSynchronization
         return await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attribute)
 
-    def pics_TC_TIMESYNC_2_2(self) -> list[str]:
```

Comment:
as above - leave this in for now.

Reply:
Thank you, my apologies, in the most recent changes I have restored this part of the code.